pallets/marketplace/src/lib.rs

```diff
@@ -625,44 +625,44 @@ pub mod pallet {
 
     #[pallet::error]
     pub enum Error<T> {
-        /// This function is reserved to the owner of a nft.
+        /// The caller is not the nft owner.
         NotNftOwner,
-        /// Nft is not present on the marketplace.
+        /// Nft is not for sale.
         NftNotForSale,
-        /// Yot cannot buy your own nft.
+        /// You cannot buy your own nft.
         NftAlreadyOwned,
-        /// Used wrong currency to buy an nft.
+        /// Currency not supported for buy this the nft.
         WrongCurrencyUsed,
-        /// We do not have any marketplace ids left, a runtime upgrade is necessary.
+        /// A marketlace id owerflow occured  (runtime upgrade necessary).
         MarketplaceIdOverflow,
         /// No marketplace found with that Id.
         UnknownMarketplace,
-        /// Commission fee cannot be more then 100.
-        InvalidCommissionFeeValue,
-        /// This function is reserved to the owner of a marketplace.
+        /// Commission fee cannot be higher than 100.
+        HighCommissionFeeValue,
```

Comment:
I would rather go with `invalid` since it tells you that the given value its not valid.
Saying that the input value is "high" doesn't mean anything relevant expect that it might have eaten too much brownies.

Reply:
`InvalidComissionFeeGiven`